cluster-provision/k8s/1.20/conformance.json

```diff
@@ -1 +1,95 @@
+{
+  "Description": "DEFAULT",
+  "UUID": "c3bc7d76-6ce8-4c8a-8bcb-5c7ae5fb22a3",
+  "Version": "v0.20.0",
+  "ResultsDir": "/tmp/sonobuoy",
+  "Resources": [
+    "apiservices",
+    "certificatesigningrequests",
+    "clusterrolebindings",
+    "clusterroles",
+    "componentstatuses",
+    "configmaps",
+    "controllerrevisions",
+    "cronjobs",
+    "customresourcedefinitions",
+    "daemonsets",
+    "deployments",
+    "endpoints",
+    "ingresses",
+    "jobs",
+    "leases",
+    "limitranges",
+    "mutatingwebhookconfigurations",
+    "namespaces",
+    "networkpolicies",
+    "nodes",
+    "persistentvolumeclaims",
+    "persistentvolumes",
+    "poddisruptionbudgets",
+    "pods",
+    "podlogs",
+    "podsecuritypolicies",
+    "podtemplates",
+    "priorityclasses",
+    "replicasets",
+    "replicationcontrollers",
+    "resourcequotas",
+    "rolebindings",
+    "roles",
+    "servergroups",
+    "serverversion",
+    "serviceaccounts",
+    "services",
+    "statefulsets",
+    "storageclasses",
+    "validatingwebhookconfigurations",
+    "volumeattachments"
+  ],
+  "Filters": {
+    "Namespaces": ".*",
+    "LabelSelector": ""
+  },
+  "Limits": {
+    "PodLogs": {
+      "Namespaces": "",
+      "SonobuoyNamespace": true,
+      "FieldSelectors": [],
+      "LabelSelector": "",
+      "Previous": false,
+      "SinceSeconds": null,
+      "SinceTime": null,
+      "Timestamps": false,
+      "TailLines": null,
+      "LimitBytes": null,
+      "LimitSize": "",
+      "LimitTime": ""
+    }
+  },
+  "QPS": 30,
+  "Burst": 50,
+  "Server": {
+    "bindaddress": "0.0.0.0",
+    "bindport": 8080,
+    "advertiseaddress": "",
+    "timeoutseconds": 10800
+  },
+  "Plugins": [
+    {
+      "name": "e2e"
+    },
+    {
+      "name": "systemd-logs"
+    }
+  ],
```

Comment:
Please change the plugins array to be empty, we controll which plugins to run from `check-cluster-up.sh`
https://github.com/kubevirt/kubevirtci/commit/d0fe64f8246e4a10b58f05ed9ab77e8743dc735f

Reply:
Thank you, I missed this change. Done